modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/portlet/action/ViewMVCRenderCommand.java

```diff
@@ -29,7 +39,52 @@ public class ViewMVCRenderCommand implements MVCRenderCommand {
 	public String render(
 		RenderRequest renderRequest, RenderResponse renderResponse) {
 
+		_httpServletRequest = _portalUtil.getHttpServletRequest(renderRequest);
+		_liferayPortletRequest = _portalUtil.getLiferayPortletRequest(
+			renderRequest);
+		_liferayPortletResponse = _portalUtil.getLiferayPortletResponse(
+			renderResponse);
+
+		StyleBookDisplayContext styleBookDisplayContext =
+			new StyleBookDisplayContext(
+				_httpServletRequest, _liferayPortletRequest,
+				_liferayPortletResponse);
+
+		StyleBookManagementToolbarDisplayContext
+			styleBookManagementToolbarDisplayContext =
+				new StyleBookManagementToolbarDisplayContext(
+					_httpServletRequest, _liferayPortletRequest,
+					_liferayPortletResponse, _cetManager,
+					_frontendTokenDefinitionRegistry,
+					styleBookDisplayContext.
+						getStyleBookEntriesSearchContainer());
+
+		renderRequest.setAttribute(
+			StyleBookManagementToolbarDisplayContext.class.getName(),
+			styleBookManagementToolbarDisplayContext);
+
+		renderRequest.setAttribute(
+			StyleBookDisplayContext.class.getName(), styleBookDisplayContext);
```

Comment:
Move this to before declaring `StyleBookManagementToolbarDisplayContext`. The idea is 
1. Declare
2. Use
3. Declare
4. Use